src/components/nodes/adm/AdmNodesTable.vue

```diff
@@ -29,23 +29,33 @@ export default defineComponent({
   },
   setup() {
     const store = useStore()
+
     const admNodes = computed(() => {
       const arr = store.getters['nodes/adm']
 
       return [...arr].sort(sortNodesFn)
     })
 
+    const isAllChecked = ref(false)
+
+    const isAllNodesEnabled = !admNodes.value.some(node => node.active === false)
+    if (isAllNodesEnabled) isAllChecked.value = true
+
+    watch(isAllChecked, (value) => {
+      admNodes.value.forEach((admNode) => {
+        if (admNode && admNode.active !== value) {
+          store.dispatch('nodes/toggle', {...admNode, active: value})
+        }
+      })
+    })
```

Comment:
1. `watch` no more needed here (check previous comment). Use a function instead.

2. Add a new action `nodes/toggleAll` in Vuex module:

```js
const toggleAll = (active) => {
   store.dispatch('nodes/toggleAll', { active })
}
```

3. And now in the `<template>` instead of `v-model`:

```vue
<NodesTableHead modelValue="isAllChecked" @update:modelValue="toggleAll" />
```

Note: If you want to keep the `v-model` approach, define the `computed` with a getter and setter. Example:

https://github.com/Adamant-im/adamant-im/blob/208cb916ff263f428db6dd766fd7d292dd7ad33d/src/components/CurrencySwitcher.vue#L43-L53